components/edit_post/index.ts

```diff
@@ -42,21 +46,35 @@ function mapStateToProps(state: GlobalState) {
         canDeletePost: haveIChannelPermission(state, teamId, channelId, deletePermission),
         codeBlockOnCtrlEnter: getBool(state, Preferences.CATEGORY_ADVANCED_SETTINGS, 'code_block_ctrl_enter', true),
         ctrlSend: getBool(state, Preferences.CATEGORY_ADVANCED_SETTINGS, 'send_on_ctrl_enter'),
+        draft,
         config,
         editingPost,
+        teamId,
         channelId,
         maxPostSize: parseInt(config.MaxPostSize || '0', 10) || Constants.DEFAULT_CHARACTER_LIMIT,
         readOnlyChannel: !isCurrentUserSystemAdmin(state) && channel.name === Constants.DEFAULT_CHANNEL,
         useChannelMentions,
     };
 }
 
+// Temporarily store draft manually in localStorage since the current version of redux-persist
+// we're on will not save the draft quickly enough on page unload.
+function setDraft(key: string, value: PostDraft) {
```

Comment:
This isn't actually needed any more. I must've forgotten to remove it from the post and comment boxes after I was done testing my migration code. If you wouldn't mind also removing it from those as well, that'd be really helpful 😄